### ISSUE TITLE
onClientTrailerAttach: cancelEvent() not working

### ISSUE THREAD

**Issue by dobosmate** [multitheftauto/mtasa-blue#2119]
**Describe the bug**
If I try the onClientTrailerAttach/onTrailerAttach event with cancelEvent() to interrupt, it will not abort.

**To reproduce**

Steps to reproduce the behaviour:
1. Sit in a truck
2. Reverse to a trailer.

[trailer.zip](https://github.com/multitheftauto/mtasa-blue/files/6135434/trailer.zip)

**Expected behaviour**
nothing will happen

**Screenshots**

**Version**
1.5.8

**Additional context**

**Comment by PlatinMTA** [comment]
Some events cannot be cancelled, maybe this is one of them.

Executing detachTrailerFromVehicle() works?

**Comment by dobosmate** [comment]
In this case, it continuously couples the trailer down / up, so the player gets stuck.

**Comment by Lpsd** [comment]
Please provide example code where you use `detachTrailerFromVehicle`, otherwise we have no idea if it's an MTA bug or code issue.

**Comment by dobosmate** [comment]
Server side
```lua
local trailer = createVehicle(trailerid, x, y, z)
setElementFrozen(trailer, true)

addEventHandler("onTrailerAttach", root, function(truck)
     if getElementFrozen(truck) then
          setTimer(function()
               detachTrailerFromVehicle(truck)
          end, 50, 1)
     end
end)
```
Client side
```lua
addEventHandler("onClientTrailerAttach", root, function(truck)
     if getElementFrozen(truck) then
          setTimer(function()
               detachTrailerFromVehicle(truck)
          end, 50, 1)
     end
end)
```
i don't use two at the same time.

**Comment by PlatinMTA** [comment]
```lua

addEventHandler("onClientTrailerAttach", root, function(truck)
	detachTrailerFromVehicle(truck)
end)
```

That works just fine in my case. I don't know why you froze the vehicle or use a timer, but try that (I used a towtruck and a jester to test).

**Comment by dobosmate** [comment]
Yes, but for a roadtrain it doesn’t work with a trailer. If I interrupt on the server side what is suggested by the wiki with a timer, it constantly hooks up / down the trailer, so the player gets stuck.

**Comment by PlatinMTA** [comment]
> Yes, but for a roadtrain it doesn’t work with a trailer. If I interrupt on the server side what is suggested by the wiki with a timer, it constantly hooks up / down the trailer, so the player gets stuck.

Can confirm this issue happens even with the code I provided.

**Comment by dobosmate** [comment]
Yes, I confirm. I tried not to freeze the trailer, but there was the same error.
